### PR TITLE
[bitnami/apache] add support for IngressClassName

### PR DESCRIPTION
**Pull request by narfeta**
**Description of the change**

This PR modifies bitnami/apache adding compatibility for ingressClassName.
https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/

**Benefits**

Adds compatibility for ingressClassName preserving backward compatibility.

**Possible drawbacks**

None

**Applicable issues**

None

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/apache/values.yaml [hunk 1]
```diff
@@ -315,6 +315,11 @@ ingress:
   ## @param ingress.apiVersion Force Ingress API version (automatically detected if not set)
   ##
   apiVersion: ""
+  ## IngressClassName is used to reference the IngressClass that should be used to implement this Ingress.
```
Comment (Mauraza) [inline bitnami/apache/values.yaml:318]: Could you add @param IngressClassName? 
We have a tool to generate the README.md with these notations

**Review by Mauraza — CHANGES_REQUESTED** [review]
Hi @narfeta,

Thanks for the PR!! Could you check the comment?

File: bitnami/apache/values.yaml (hunk 1 above)
Comment (narfeta) [reply bitnami/apache/values.yaml:319]: Sorry for that, just added the @param.

**Review by Mauraza — APPROVED** [review]
Hi @narfeta,

Thanks again for the PR!!

**Pull request merged**